<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-QJLSV9M" failures="0" tests="8" name="com.api.PNvaditaions.LoginFTvalidation" time="13.921" errors="1" timestamp="2023-05-10T13:27:42 IST" skipped="0">
  <testcase classname="com.api.PNvaditaions.LoginFTvalidation" name="RestApiLoginValidations" time="0.324"/>
  <system-out>
    <![CDATA[Actual status code is =>400=>-AND-expected status code=>400
Actual Response body =>"Unexpected token } in JSON at position 98"=>Expected Response Body is=>Unexpected token } in JSON at position
Actual Response body =>application/json; charset=utf-8=>Expected Response Body is=>application/json; charset=utf-8]]>
  </system-out>
  <testcase classname="com.api.PNvaditaions.LoginFTvalidation" name="RestApiLoginValidations" time="0.213"/>
  <system-out>
    <![CDATA[Actual status code is =>400=>-AND-expected status code=>400
Actual Response body =>"Unexpected string in JSON at position 75"=>Expected Response Body is=>Unexpected string in JSON at position
Actual Response body =>application/json; charset=utf-8=>Expected Response Body is=>application/json; charset=utf-8]]>
  </system-out>
  <testcase classname="com.api.PNvaditaions.LoginFTvalidation" name="RestApiLoginValidations" time="7.587">
    <error message="
[public void com.api.PNvaditaions.LoginFTvalidation.RestApiLoginValidations(java.lang.String,java.lang.String,java.lang.String,double,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: RestApiLoginValidations([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2, double arg3, java.lang.String arg4, java.lang.String arg5])
Arguments: [null,null,null,null]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.api.PNvaditaions.LoginFTvalidation.RestApiLoginValidations(java.lang.String,java.lang.String,java.lang.String,double,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: RestApiLoginValidations([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2, double arg3, java.lang.String arg4, java.lang.String arg5])
Arguments: [null,null,null,null]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:909)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:37)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- RestApiLoginValidations -->
  <system-out/>
  <testcase classname="com.api.PNvaditaions.LoginFTvalidation" name="RestApiLoginValidations" time="4.749"/>
  <system-out>
    <![CDATA[Actual status code is =>200=>-AND-expected status code=>200
Actual Response body =>[{"jwt":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiN2MyMWM4YzEtM2VlMC01MGViLThlMDItNGRhZWExYjQ4YzEyIiwidXNlcl9jYXRlZ29yeSI6IkFkbWluIiwidXNlcl9jbGllbnRfaWQiOiJOLkEuIiwidXNlcl90eXBlIjoiTi5BLiIsImlhdCI6MTY4MzcwNTQ2Mn0.SOcnoaRXu1q96gi_YnSFcu5MO3FgGpls7pSQqG6sI1w"}]=>Expected Response Body is=>eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.
Actual Response body =>application/json; charset=utf-8=>Expected Response Body is=>application/json; charset=utf-8]]>
  </system-out>
  <testcase classname="com.api.PNvaditaions.LoginFTvalidation" name="RestApiLoginValidations" time="0.288"/>
  <system-out>
    <![CDATA[Actual status code is =>400=>-AND-expected status code=>400
Actual Response body =>"Invalid Password"=>Expected Response Body is=>Invalid Password
Actual Response body =>application/json; charset=utf-8=>Expected Response Body is=>application/json; charset=utf-8]]>
  </system-out>
  <testcase classname="com.api.PNvaditaions.LoginFTvalidation" name="RestApiLoginValidations" time="0.245"/>
  <system-out/>
  <testcase classname="com.api.PNvaditaions.LoginFTvalidation" name="RestApiLoginValidations" time="0.254"/>
  <system-out/>
  <testcase classname="com.api.PNvaditaions.LoginFTvalidation" name="RestApiLoginValidations" time="0.261"/>
  <system-out>
    <![CDATA[Actual status code is =>400=>-AND-expected status code=>400
Actual Response body =>"User Email Doesn't Exists"=>Expected Response Body is=>User Email Doesn't Exists
Actual Response body =>application/json; charset=utf-8=>Expected Response Body is=>application/json; charset=utf-8]]>
  </system-out>
</testsuite> <!-- com.api.PNvaditaions.LoginFTvalidation -->
